go/2_renew-token.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	vault "github.com/hashicorp/vault/api"
+)
+
+// Once you've set the token for your Vault client, you will need to periodically renew its lease.
+// A function like this should be run as a goroutine to avoid blocking.
+func renewToken(client *vault.Client) {
```

Comment:
Do we want a comment here suggesting that a production app might want to be tolerant of intermittent failures and retry rather than exiting?

Reply:
We could also reference https://www.vaultproject.io/docs/enterprise/consistency for ent users, and note that without applying one of those techniques, they should expect occasional issues using a fresh token if perf standbys handle requests.

Reply:
Good suggestions, and now I finally had the reason to read up on eventual consistency with Vault. I'll add those comments.